server.js

```diff
@@ -1,22 +1,26 @@
-import {createRequire} from "module";
+import { createRequire } from "module";
 const require = createRequire(import.meta.url);
+// load up env variables from the .env file
+require('dotenv').config()
 
 const http = require('http');
 const fs = require('fs');
 const url = require('url');
 const figlet = require('figlet');
 
-import {fileTypes} from "./file.js";
+import { fileTypes } from "./file.js";
+import { _parse } from "./api.js";
 
 
-const server = http.createServer((req, res) =>{
+const server = http.createServer((req, res) => {
     // if protocol is undefined, default to http for now
     const base = ((req.protocol)? req.protocol : "http") + '://' + req.headers.host + '/'
     const fullURL =  new url.URL(req.url, base)
     const page = fullURL.pathname
     const params = fullURL.searchParams
     if (page === "/api"){
         // TODO: handle api here
+        jsonSend(undefined, params, res)
```

Comment:
```suggestion
        // TODO
```
This function is not in the code anymore, should be deleted and replaced with a comment for now.